backend/typescript/services/implementations/camperService.ts

```diff
@@ -302,6 +302,7 @@ class CamperService implements ICamperService {
         contactEmail: waitlistedCamper.contactEmail,
         contactNumber: waitlistedCamper.contactNumber,
         campSession: waitlistedCamper.campSession,
+        linkExpiry: undefined,
```

Comment:
do we need this if linkExpiry is optional

Reply:
I thought it would be nice so they know its a field, but I guess not